src/JMS/Serializer/EventDispatcher/Subscriber/SymfonyValidatorSubscriber.php

```diff
@@ -21,14 +21,19 @@
 use JMS\Serializer\EventDispatcher\Event;
 use JMS\Serializer\EventDispatcher\EventSubscriberInterface;
 use JMS\Serializer\Exception\ValidationFailedException;
-use Symfony\Component\Validator\ValidatorInterface;
 
 class SymfonyValidatorSubscriber implements EventSubscriberInterface
 {
     private $validator;
 
-    public function __construct(ValidatorInterface $validator)
+    public function __construct($validator)
     {
+        if (!$validator instanceof \Symfony\Component\Validator\Validator\ValidatorInterface // 3.x
```

Comment:
@stof This was the only way I could think of to maintain 2.3 compatibility with the new Validator structure. Well, easily, at least. Not a big fan. Thoughts?